frontend/Components/ListingCard/ListingCard.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Card } from "antd";
+import ListingCardStyles from "./ListingCard.module.css";
```

Comment:
Por lo general, se usa de esta forma

```suggestion
import styles from "./ListingCard.module.css";
```